PackageBuilder/PackageBuilder.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFrameworks>net6.0;net8.0</TargetFrameworks>
```

Comment:
Simply replacing with `<TargetFramework>net6.0</TargetFramework>` will break comaptibility with self-hosted runners without .NET 8.

This repository doesn't have versioning so we cannot make breaking change so I added support for .net 8 instead of replading with .NET 8.